pallets/gear/Cargo.toml

```
@@ -46,6 +46,7 @@ wabt = "0.10"
 gear-core = { path = "../../core" }
 hex-literal = "0.3.4"
 tests-distributor = { path = "../../tests/distributor" }
+tests-init-wait = { path = "../../tests/init_wait" }
```

```suggestion
tests-init-wait = { path = "../../tests/init-wait" }
```

and corresponding changes

---

Sorry didn't understand your suggestion

---

hyphen!